src/jsdoc.ts

```diff
@@ -177,14 +166,29 @@ export function parse(comment: string): {tags: Tag[], warnings?: string[]}|null
       let tag: Tag = {tagName};
       if (parameterName) tag.parameterName = parameterName;
       if (text) tag.text = text;
+      if (!arrayIncludes(JSDOC_TAGS_WHITELIST, tagName)) {
+        // Escape tags we don't understand.  This is a subtle
+        // compromise between multiple issues.
+        // 1) If we pass through these non-Closure tags, the user will
+        //    get a warning from Closure, and the point of tsickle is
+        //    to insulate the user from Closure.
+        // 2) The output of tsickle is for Closure but also may be read
+        //    by humans, for example non-TypeScript users of Angular.
+        // 3) Finally, we don't want to warn because users should be
+        //    free to add whichever JSDoc they feel like.  If the user
+        //    wants help ensuring they didn't typo a tag, that is the
+        //    responsibility of a linter.
+        tag.escape = true;
```

Comment:
Rather than adding an attribute here, you can just look it up down when we call tagToString.